bin/apisix

```diff
@@ -1000,7 +1000,7 @@ local function init_etcd(show_output)
             os.exit(1)
         end
 
-        local cluster_version = body["etcdcluster"]
+        cluster_version = body["etcdcluster"]
```

Comment:
Maybe delete the outside definition is better, since we use the new one in each cycle.

Reply:
agree +1